### PR TITLE
Remove fiat dropdown for MVP

### PR DESCRIPTION
**Pull request by Sonicrida**
### Changes

* Removes the fiat dropdown from the dashboard

### Screenshots

image

### CODE REVIEW

File: common/v2/features/Dashboard/components/WalletBreakdown.tsx [hunk 1]
```diff
@@ -76,11 +76,6 @@ function WalletBreakdown() {
   return (
     <div className="WalletBreakdown">
       <div className="WalletBreakdown-selectWrapper">
-        <ComboBox
-          className="WalletBreakdown-selectWrapper-select"
```
Comment (nickmccurdy) [inline common/v2/features/Dashboard/components/WalletBreakdown.tsx]: Please remove the relevant SCSS

**Comment by nickmccurdy** [comment]
https://app.clubhouse.io/mycrypto/story/1321

File: common/v2/features/Dashboard/components/WalletBreakdown.tsx (hunk 1 above)
Comment (Sonicrida) [reply common/v2/features/Dashboard/components/WalletBreakdown.tsx]: this is still used for the accounts dropdown actually

**Comment by nickmccurdy** [comment]
Thanks, this is better.

On a wide (1440p) screen the proportions look better:
image

But on a skinner screen it's proportionately a lot wider:
image

Do we need this extra width on smaller screens or can we make it relative to the dashboard column's width?

**Pull request merged**